### ISSUE TITLE
allow for multi-line commands

### ISSUE THREAD

**Issue by noonien** [droundy/fac#25]
something of the sorts would be really helpful, and would help not require having to write too many build scripts, or have too long lines.
```
| mkdir -p build/src
| for f in src/*; do
|   echo  "| gcc -I../include/ -Wall -Werror -pedantic -std=gnu18 -c \"../$f\" -o \"${f%.c}.o\""
| done | tee build/.fac
```

Perhaps `| ` would not work since it seems to not require a blank line between rules, and this might break old facfiles.

**Comment by droundy** [comment]
Something like that could certainly work. It would not work so well for a portable build script, but since fac isn't very portable that isn't a steering argument against.

I agree that '|' can't be the continuation character for the reason you give. In fact your example would end up creating one rule that compiles all the files, since you didn't add blank spaces between the rules.

On the other hand, you can achieve the same outcome by using semicolons in your shell command, and then you would be able to copy and paste your rule into a shell to test it out...

**Comment by noonien** [comment]
Yes, I've currently resorted to using semicolons, but it makes reading the .fac harder, especially for more commands.
